### ISSUE TITLE
UI Events in Android 

### ISSUE THREAD

**Issue by utkarsh006** [utkarsh006/Eazy-Android#34]
- Explain about UI, UI Events and User Events.
- Explain User Event Decision tree.
- Cover the use cases.
- Do explain the topic in layman terms along with suitable example.

**Comment by devashishpant89** [comment]
I would like to work on this. Could you please assign it to me?

**Comment by abhishekpathak1** [comment]
I want to work on this. Please assign it to me!!

**Comment by vinitfreakk** [comment]
would like to contribute in this

**Comment by abhishekpathak1** [comment]
yes.
